pkg/tools/ippool/ippool.go

```diff
@@ -230,6 +230,41 @@ func (tree *IPPool) Pull() (net.IP, error) {
 	return ipFromIPAddress(ip, tree.ipLength), nil
 }
 
+// PullIPString - returns requested IP address from the pool by string
+func (tree *IPPool) PullIPString(ipString string, exclude ...*IPPool) (*net.IPNet, error) {
+	ip, _, err := net.ParseCIDR(ipString)
+	if err != nil {
+		return nil, err
+	}
+
+	return tree.PullIP(ip, exclude...)
+}
+
+// PullIP - returns requested IP address from the pool
+func (tree *IPPool) PullIP(ip net.IP, exclude ...*IPPool) (*net.IPNet, error) {
```

Comment:
I'm not quite following `PullIP(...)` operation. Could you clarify a bit why we need it?

Reply:
This was added for consistency with other pool methods - most of them have 2 versions: `Name(...)` and `NameString(...)`